.gitlab-ci.yml

```diff
@@ -1,32 +1,41 @@
 image: tmaier/docker-compose:latest
 
+stages:
+  - test
+  - deploy
+
 services:
   - docker:dind
 
 before_script:
   - docker info
   - docker-compose --version
-  - docker login -u "$CI_REGISTRY_USER" -p "$CI_REGISTRY_PASSWORD" $CI_REGISTRY
-
-build_backend:
-  stage: build
-  tags:
-    - docker
-  script:
-    - docker-compose build
```

Comment:
Bardzej pytanie niż recenzja, czemu w CI usunięty jest build? Nie mogę znaleźć gdzie jest przesunięty, czy to inaczej działa?

Reply:
Build będzie w testach z tego co rozmawiałem z szymonem. Nie robi to nam dużej różnicy a będzie szybciej CI działać + nie bedziemy musieli tyle zasobow miec na to